src/webgpu/api/validation/encoding/cmds/copyTextureToTexture.spec.ts

```diff
@@ -510,6 +512,51 @@ g.test('copy_within_same_texture')
     );
   });
 
+// TODO(jiawei.shao@intel.com): test 'stencil-only' is also valid for the texture format 'stencil8'
+g.test('copy_aspects')
```

Comment:
This can be a follow-up, but let's please adopt the [new style](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) and have a description for each test.